### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.3 - es. 3 (7 ottobre)

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/calcolo-delle-probabilita#274] [Esercizi M-Z]
![es.3](https://github.com/user-attachments/assets/e1c82c89-fb22-44b4-8f52-79808cd43935)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/8d911926-2dc3-41c6-b588-f539c0459e52)
